recipes/ocl-icd-system/activate.sh

```diff
@@ -0,0 +1,9 @@
+conda_ocl_icd_system_activate () {
```

Comment:
Can you make this a `post-link.sh` script and the deactivate a `pre-unlink.sh` script.